### ISSUE TITLE
[universal-redux] Webfonts, how to?

### ISSUE THREAD

**Issue by 0xsven** [diegohaz/arc#12]
My first thought was to use `injectGlobal` from styled-components in the App.js file and copying the font to the `./public` folder. 

```
injectGlobal`
  @font-face {
    font-family: 'Avenir';
    src:  url('/avenir-roman.woff2') format('woff2'),
          url('/avenir-roman.woff') format('woff');
    font-weight: 400;
    font-style: normal;
  }
`
```
(Outside the component, see #11)

This somehow seems to cause problems in chrome (not in safari and FF though). Let me try to explain:

1. When opening my index route (rendered on the server) the browser loads the font (initiated from the index html file) and displays the text correctly.
2. Then I click on a link that leads to a different route: injectGlobal from app.js runs and the font gets downloaded again (because the initiator is app.js this time). The result is that chrome renders all text that uses the font as transparent.

This seems like a bug in chrome. Is there a better way to load fonts? 
(I guess falling back to the styles.css file is always an option)

**Comment by diegohaz** [comment]
Hi, @0xsven 

I couldn't reproduce it. Can you provide a repo with the issue?

**Comment by 0xsven** [comment]
@diegohaz Here you go:

https://github.com/0xsven/arc-temp

Simple run `npm run dev` open the index route in a browser. Then change the route by clicking the sample page link. It looks like this on my end: https://www.evernote.com/l/ACXsmztlg71HIrU_YmZLpkZJI22LlFR18pgB/image.png

**Comment by diegohaz** [comment]
Yeah, I can reproduce it now. It might be related to `styled-components` on server side (I also tried it on the redux branch, without ssr, but it works there).

It seems `styled-components` is replacing the styles rendered on the server, then causing the re-download.

I will continue to look for a solution. If the styles.css works, I suggest you to use that for now.

**Comment by diegohaz** [comment]
Fixed in [styled-components v1.4.3](https://github.com/styled-components/styled-components/blob/master/CHANGELOG.md#v143---2017-02-04) 🎉
